legend-engine-xt-sql-postgres-server/src/main/java/org/finos/legend/engine/postgres/ExecutionDispatcher.java

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package org.finos.legend.engine.postgres;
+
+import org.finos.legend.engine.language.sql.grammar.from.antlr4.SqlBaseParser;
+import org.finos.legend.engine.language.sql.grammar.from.antlr4.SqlBaseParserBaseVisitor;
+import org.finos.legend.engine.postgres.handler.SessionHandler;
+import org.finos.legend.engine.postgres.handler.empty.EmptySessionHandler;
+import org.finos.legend.engine.protocol.sql.metamodel.QualifiedName;
+
+import java.util.List;
+
+public class ExecutionDispatcher extends SqlBaseParserBaseVisitor<SessionHandler>
+{
+    private static final TableNameExtractor EXTRACTOR = new TableNameExtractor();
+    private static final SessionHandler EMPTY_SESSION_HANDLER = new EmptySessionHandler();
+    private final SessionHandler dataSessionHandler;
+    private final SessionHandler metaDataSessionHandler;
+
+    public ExecutionDispatcher(SessionHandler dataSessionHandler, SessionHandler metaDataSessionHandler)
+    {
+        this.dataSessionHandler = dataSessionHandler;
+        this.metaDataSessionHandler = metaDataSessionHandler;
+    }
+
+    @Override
+    public SessionHandler visitSet(SqlBaseParser.SetContext ctx)
+    {
+        return EMPTY_SESSION_HANDLER;
+    }
+
+    @Override
+    public SessionHandler visitDefault(SqlBaseParser.DefaultContext ctx)
```

Comment:
perhaps helpful to note that Default corresponds to "Select"

Reply:
added comment